src/main/java/bio/terra/service/snapshotbuilder/query/Query.java

```diff
@@ -102,6 +108,10 @@ public String renderSQL() {
       sql += " " + having.renderSQL();
     }
 
+    if (limit != null) {
+      sql += " LIMIT " + limit;
```

Comment:
So, it turns out that `LIMIT` doesn't work this way in T-SQL (which is what azure/synapse is). In T-SQL, you have to use `TOP N SELECT ...` whereas big query uses `SELECT ... LIMIT N`.

In the prototype I addressed this by passing in `platform` to `Query.renderSQL()`. https://github.com/DataBiosphere/jade-data-repo/blob/3006e94104e1ee000adb6876504749ee8223a499/src/main/java/bio/terra/tanagra/query/Query.java#L37